src/components/Intro.vue

```diff
@@ -8,47 +8,18 @@
     <!-- TAKEAWAY TITLE -->
     <template v-slot:takeAway>
       <h2>
-        A <a
-          href="https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2017GL073551"
-          target="_blank"
-        >majority</a> of the water in the western U.S. comes from snowmelt.
+        A majority of the water in the western U.S. comes from snowmelt<sup>1</sup>.
```

Comment:
what's <sup>?

Reply:
OH MY GOD COOL ^ it just made the question mark a superscript, I had no idea that that was an HTML tag!

Reply:
lolololol. I did that for citations. It's not the most glamorous way to do it, but it's legit. We can always style a little better clsoer to release